configs/sparse_rcnn/README.md

```diff
@@ -0,0 +1,28 @@
+# Sparse R-CNN: End-to-End Object Detection with Learnable Proposals
+
+## Introduction
+
+[ALGORITHM]
+
+```
+@article{peize2020sparse,
+  title   =  {{SparseR-CNN}: End-to-End Object Detection with Learnable Proposals},
+  author  =  {Peize Sun and Rufeng Zhang and Yi Jiang and Tao Kong and Chenfeng Xu and Wei Zhan and Masayoshi Tomizuka and Lei Li and Zehuan Yuan and Changhu Wang and Ping Luo},
+  journal =  {arXiv preprint arXiv:2011.12450},
+  year    =  {2020}
+}
+```
+
+## Results and Models
+
+| Model        | Backbone  | Style   | Lr schd | Number of Proposals |Multi-Scale| RandomCrop  | box AP  | Config | Download |
+|:------------:|:---------:|:-------:|:-------:|:-------:            |:-------: |:---------:|:------:|:------:|:--------:|
+| Sparse R-CNN | R-50-FPN  | pytorch | 1x      |   100               | False     |  False     |  38.2  |         |       |
+| Sparse R-CNN | R-50-FPN  | pytorch | 3x      |   100               | True     |   False     |  42.8  |         |       |
+| Sparse R-CNN | R-50-FPN  | pytorch | 3x      |   300               | True      |  True      |  45.0  |         |       |
+| Sparse R-CNN | R-101-FPN | pytorch | 3x      |   100               | True      |  False     |  44.2  |         |       |
+| Sparse R-CNN | R-101-FPN | pytorch | 3x      |   100               | True      |  True      |  46.2  |         |       |
+
+### Notes
+
+We observe about 0.3 AP noise especially when using Res-101 as the backbone.
```

Comment:
Can we report the average results which are more reproducible?

Reply:
I only conduct the experiment of res101 twice, each time consume about three days.